crates/dataflow-jit/src/ir/types.rs

```diff
@@ -0,0 +1,393 @@
+use crate::{
+    codegen::NativeType,
+    ir::{function::InputFlags, LayoutId, RowLayoutCache},
+};
+use bitvec::vec::BitVec;
+use std::fmt::{self, Debug, Display, Write};
+
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash)]
+pub enum ColumnType {
+    Bool,
+    U16,
```

Comment:
How come constants start with 32 bits but there are 16-bit types?

Reply:
Just because I haven't fully built all of that out, hasn't been necessary yet